public/server/db/ps.json

```diff
@@ -0,0 +1,5 @@
+{
+    "definition": "Problem Statement",

```

Comment:
Thinking hard about this one... but I really think it could go too.

Reply:
Alright NP, As you wish 🙃.